pybotvac/robot.py

```diff
@@ -241,14 +242,11 @@ def __init__(self, serial, secret):
         self.secret = secret
 
     def __call__(self, request):
-        # Due to https://github.com/stianaske/pybotvac/issues/30
-        # Neato expects and supports authentication header ONLY for en_US
-        saved_locale = locale.getlocale(locale.LC_TIME)
-        locale.setlocale(locale.LC_TIME, 'en_US.utf8')
-        
-        date = time.strftime('%a, %d %b %Y %H:%M:%S', time.gmtime()) + ' GMT'
-
-        locale.setlocale(locale.LC_TIME, saved_locale)
+        # We have to format the date according to RFC 2616
+        # https://tools.ietf.org/html/rfc2616#section-14.18
+
+        now = datetime.now(timezone.utc)
+        date = format_datetime(now, True)
```

Comment:
yeah this is perfect!

https://docs.python.org/3/library/email.utils.html#email.utils.format_datetime

> ... usegmt may be set to True, in which case the string GMT is used instead of the numeric timezone offset. This provides a way to generate standards conformant HTTP date headers.